### PR TITLE
cmd/scollector: Add support for MSSQL Named Instances

### PR DESCRIPTION
**Pull request by gbrayut**
This is based off of #1579 but is rebased on current master, fixes some whitespace issues, and squashes to a single commit.

@captncraig or @kylebrandt  please review. Currently running on a few systems an sending data to ny-devbosun (see http://goo.gl/ZZjgKX )

closes #1579

### CODE REVIEW

File: cmd/scollector/collectors/sql_windows.go [hunk 1]
```diff
@@ -85,24 +91,32 @@ func c_mssql() (opentsdb.MultiDataPoint, error) {
...
-		Add(&md, "mssql.logins", v.LoginsPersec, nil, metadata.Counter, metadata.PerSecond, descMSSQLLoginsPersec)
-		Add(&md, "mssql.logouts", v.LogoutsPersec, nil, metadata.Counter, metadata.PerSecond, descMSSQLLogoutsPersec)
-		Add(&md, "mssql.mars_deadlocks", v.MarsDeadlocks, nil, metadata.Counter, metadata.Count, descMSSQLMarsDeadlocks)
-		Add(&md, "mssql.proc_blocked", v.Processesblocked, nil, metadata.Gauge, metadata.Count, descMSSQLProcessesblocked)
-		Add(&md, "mssql.temptables_created", v.TempTablesCreationRate, nil, metadata.Counter, metadata.PerSecond, descMSSQLTempTablesCreationRate)
-		Add(&md, "mssql.temptables_to_destroy", v.TempTablesForDestruction, nil, metadata.Gauge, metadata.Count, descMSSQLTempTablesForDestruction)
-		Add(&md, "mssql.transactions_total", v.Transactions, nil, metadata.Gauge, metadata.Count, descMSSQLTransactions)
+	for _, w := range svc_dst {
+		var dst []Win32_PerfRawData_MSSQLSERVER_SQLServerGeneralStatistics
+		var q = wmi.CreateQuery(&dst, `WHERE Name <> '_Total'`)
```
Comment (kylebrandt) [inline cmd/scollector/collectors/sql_windows.go]: Would use `q :=` and `label :=`. Generally var isn't used if you are assigning a value.

File: cmd/scollector/collectors/sql_windows.go [hunk 2]
```diff
@@ -499,3 +533,8 @@ type MSCluster_Resource struct {
 	Type       string
 	State      uint32
 }
+
+func instance_wmiquery(instancename string, wmiquery string) string {
```
Comment (kylebrandt) [inline cmd/scollector/collectors/sql_windows.go]: Other than collectors (and that is just a bad convention) funcs should be instanceWMIQuery, unless there is something very similar and you are following the pattern.

**Comment by kylebrandt** [comment]
Other than the cosmetic comments LGTM

**Comment by gbrayut** [comment]
Thanks. Amended fixes for Kyle's comments. Will check with Craig about merging tomorrow.

**Pull request closed without merging**